crates/sui-move-build/src/lib.rs

```diff
@@ -705,10 +705,14 @@ impl CompiledPackage {
 
 /// Create a set of [Dependencies] from a [SystemPackagesVersion]; the dependencies are override git
 /// dependencies to the specific revision given by the [SystemPackagesVersion]
+///
+/// Skips "Deepbook" dependency.
 pub fn implicit_deps(packages: &SystemPackagesVersion) -> Dependencies {
+    let blacklist = vec!["Deepbook".to_string()];
     packages
         .packages
         .iter()
+        .filter(|package| !blacklist.contains(&package.package_name))
```

Comment:
Let's just do the simple thing for now.

```suggestion
    packages
        .packages
        .iter()
        .filter(|package| package.package_name != "Deepbook")
```

Otherwise, `s/blacklist/blocklist/`.